packages/deployment/app/config.yml

```diff
@@ -4,8 +4,10 @@ default:
 
   urs_url: https://uat.urs.earthdata.nasa.gov
 
-
-  execution_payload_retention_period: 30 ## days
+  non_complete_execution_payload_timeout: 30
```

Comment:
It might be common sense, but I kind of liked seeing the `# days` comment. Completely up to you, though.

Reply:
I'm not sure config values fall under the domain of common sense.    The code itself explains what the values actually are, and the user-facing docs define what the units are, this felt like another place to miss a comment update.... that said, if you think it's useful to have there for reference, I have no strong feelings either way. 

185d5018